Resources/config/imagine.xml

```diff
@@ -84,12 +84,12 @@
 
         <!--  Templating helpers and extensions -->
 
-        <service id="liip_imagine.twig.extension" class="Liip\ImagineBundle\Templating\ImagineExtension" public="false">
+        <service id="liip_imagine.templating.filter_extension" class="Liip\ImagineBundle\Templating\FilterExtension" public="false">
             <tag name="twig.extension" />
-            <argument type="service" id="liip_imagine.cache.manager" />
+            <argument type="service" id="liip_imagine.templating.filter_helper" />
```

Comment:
The intention was to remove code duplication; now that we've bumped the PHP requirements, though, traits serve the same purpose, so this has been rectified in that was, allowing both the helper and extension to again consume the `CacheManager`.